### PR TITLE
Use linux-headers for current running kernel

### PR DESCRIPTION
**Pull request by warrenvw**
Resolve build failure at https://circleci.com/gh/StackStorm/packer-st2/963.

### CODE REVIEW

File: .circleci/ansible/bin/install_dependencies.sh [hunk 1]
```diff
@@ -11,6 +11,6 @@ if [ ! -e /sbin/vboxconfig ]; then
   sudo sh -c "echo 'deb http://download.virtualbox.org/virtualbox/debian xenial contrib non-free' > /etc/apt/sources.list.d/virtualbox.list"
   wget -q http://download.virtualbox.org/virtualbox/debian/oracle_vbox.asc -O- | sudo apt-key add -
   sudo apt-get update && sudo apt-get install -y --allow-unauthenticated virtualbox-5.2
-  sudo apt-get install -y linux-headers-generic linux-headers-4.4.0-112-generic
+  sudo apt-get install -y linux-headers-generic linux-headers-$(uname -r)
```
Comment (armab) [inline .circleci/ansible/bin/install_dependencies.sh]: That makes sense now :+1: 

But shouldn't headers install happen before installing virtualbox?

Comment (warrenvw) [reply .circleci/ansible/bin/install_dependencies.sh]: Likely. I wanted to try this change first. I'll change the order in the next commit.

Comment (armab) [reply .circleci/ansible/bin/install_dependencies.sh]: :100: That's a good way of doing it!

Comment (warrenvw) [reply .circleci/ansible/bin/install_dependencies.sh]: The `linux-headers` are not required to install `virtualbox-5.2`.

Comment (armab) [reply .circleci/ansible/bin/install_dependencies.sh]: Why are we installing them then?

Comment (warrenvw) [reply .circleci/ansible/bin/install_dependencies.sh]: It fails if I remove that line. See 0e4cff1. I'll now try with it prior to the installation of `virtualbox-5.2`, as discussed yesterday.

**Pull request merged**